src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -329,23 +326,13 @@ RoundaboutType RoundaboutHandler::getRoundaboutType(const NodeID nid) const
         if (1 == roundabout_name_ids.size() &&
             0 == connected_names.count(*roundabout_name_ids.begin()))
             return RoundaboutType::Rotary;
-        else
-            return RoundaboutType::Roundabout;
     }
 
-    if (radius <= MAX_ROUNDABOUT_INTERSECTION_RADIUS)
```

Comment:
There were a few cucumber cases testing roundabout sizes. They have been adjusted for this change (see above).